packages/loader/container-loader/README.md

```diff
@@ -79,7 +87,20 @@ When container is closed, the following is true (in no particular order):
 3. "readonly" event fires on DeltaManager & Container (and Container.readonly property is set to true)  indicating to all data stores that container is read-only, and data stores should not allow local edits, as they are not going to make it.
 4. "disconnected" event fires, if connection was active at the moment of container closure.
 
-`"closed"` event is available on Container for hosts. `"disposed"` event is delivered to container runtime when container is closed. But container runtime can be also disposed when new code proposal is made and new version of the code (and container runtime) is loaded in accordance with it.
+`"closed"` event is available on Container for hosts.
+
+#### `Container.dispose()`
+
+Once disposed, container terminates connection to ordering service, and any local changes (former or future) do not propagate to storage. This method is to be used when the container is **NOT** expected to be used anymore.
+
+When container is disposed, the following is true (in no particular order):
+
+1. Container.closed property is set to true
+2. "disposed" event fires on container
```

Comment:
It's surprising that one event is called "disposed" and the other "dispose".  I agree "disposed" is better, but I'd vote for consistency instead and go with "dispose"